Makefile

```diff
@@ -0,0 +1,15 @@
+MARATHON_TAG := v1.5.5
+
+.PHONY: build-image
+build-image:
+	docker build -t marathon-build .
+
+.PHONY: build-package
+build-package: build-image
+	git clone https://github.com/mesosphere/marathon.git
+	cd marathon && git checkout ${MARATHON_TAG}
```

Comment:
this is probably a good use-case for git submodules /shrug

Reply:
That's a good point, although I prefer having the tagged version in the Makefile instead of updating the submodule each time (and then only the commit Sha at the end of the tag can be actually used?). That and the complexity of working with submodules makes me think that cloning it each time is probably nicer (although then it needs to be cleaned before building again), but I feel it's a bit more straightforward at least.